gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```

Comment:
@nepeat can you give a step-by-step to reproduce?  I've run both `gulp lint` and `gulp server` and haven't run into to issue you describe.

:-1: on this line change, it's inelegant and shouldn't be necessary.